### DISCUSSION TITLE
Add clipboard option, like gboard #

### DISCUSSION THREAD

**Discussion by Swyzy-dotcom** [AnySoftKeyboard/AnySoftKeyboard#2692] [Ideas]
Add clipboard option, like gboard #

**Comment by ghost** [comment]
this would be nice, I would like an encrypted clipboard outside of the OS level. Don't want other apps getting in there

**Comment by menny** [comment]
Can you give me more information about that? What is the user-experience with gboard's clipboard?

**Reply by 3xploiton3** [reply]
we can fill field like address, email, or number phone without having typing again
